int8        waypoint_id
---
float64     distance